Audible-Altimeter-Embedded/audio/private/rpi_i2s_audio_driver.cpp

```diff
@@ -0,0 +1,45 @@
+
+#include "hardware/dma.h"
+#include "pio_serializer.pio.h"
```

Comment:
What is this file?

Reply:
oh all the `*.pio` files get consumed by cmake using `pico_generate_pio_header` which generates a `*.pio.h` file. In this case, `pio_serializer.pio` was the old name of the `i2s.pio`. I'll update it. I'll need to investigate if it's possible for the pico sdk to generate the header with an `.hpp` extension, maybe I can write a little cmake function to rename the file once it's created.